docs/en/getting-started/create-or-restore-your-status-profile-using-a-recovery-phrase.md

```diff
@@ -1,16 +1,97 @@
 ---
 id: 374
 revision: 0
+revision: o
 language: en
 hide:
   - navigation
 ---
 
 # Create or restore your Status profile using a recovery phrase
 
- :octicons-tools-24: In progress
+You can link your existing Ethereum address to a new Status profile. For instance, if you already have a [non-custodial Ethereum wallet][how-to-use-wallet-your-quick-start-guide] such as Metamask or Trust Wallet, you already have an Ethereum address and a [recovery phrase][understand-your-status-keys-and-recovery-phrase].
 
-!!! note ""
-     We're working on this content.
+## What to expect
 
---8<-- "includes/urls-en.txt"
\ No newline at end of file
+- Your wallet funds are safe and recoverable as long as you know your recovery phrase and [keep it safe][back-up-and-secure-your-recovery-phrase].
+- You can create a new Status profile using your existing Ethereum address.
```

Comment:
This point seems repeated. Because part of the task is to create a new profile using the recovery phrase of an existing Ethereum address.

Reply:
Removed and replaced with the previous suggestion about the Status Wallet.